backend/mcp_backend/integration/crew_mcp_integration.py

```
@@ -0,0 +1,634 @@
+                    process=Process.sequential,
+                    verbose=True,
+                )
+
+                # Execute the crew
+                logger.info("Executing CrewAI crew...")
+                result = time_crew.kickoff()
+                logger.info("CrewAI crew execution completed successfully")
+                return str(result)
+
+        except Exception as e:
+            logger.error(f"Error in CrewAI MCP integration: {e}")
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)